src/labels/CustomLabels.labels

```diff
@@ -7028,6 +7028,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Label for the Template Name field</shortDescription>
         <value>Template Name</value>
     </labels>
+    <labels>
+        <fullName>gePaymentRequestCardholderName</fullName>
```

Comment:
I guess it is fine for the label to belong to GE labels (not `common`) even though it will be used on both forms.
@sam-knox , @melissabarber : can you please review the new label?

Reply:
I thought about moving to RD2, but it seems so specific to "gifts" in general (vs. Common for things that are very generic) so I ended up leaving it as a "ge" label.

Reply:
Thanks for the heads up about this. Thanks for leaving it as ge, but definitely something to keep in mind for components like this that will be shared across GE and RD entry screens. FYI, @ptewson-sfdo @psadie-sfdo @vmartinov-sforg @bdvorachek @dfuller114 @kenlewis-sfdo @voduyemi

Reply:
K, let's try that again. My comment didn't make a whole lot of sense. I was operating under the assumption that this was a label we created for GE when we were setting up Elevate functionality in GE. Since it's new and it could be used by GE or RD entry screens, should this have the "common" prefix instead of GE? @psadie-sfdo do you recall why we *don't* have a label like this in GE?

Reply:
@melissabarber - we don't have a label for this because our credit card widget doesn't have a cardholder name input field. We construct the cardholder name on the fly and send it along with our elevate requests in the format of "{contact first name} {contact last name}" or "{organization account name} {organization account name}".

If I remember correctly, the sentiment behind that decision was that we wanted to eliminate the step for a gift entry user since we had all of that information already. I think we may need to re-visit that decision soon.

For the shared RD/GE label, could we consider prefixing with `npsp`? Specifically for labels like this which doesn't really fit `common`.

Reply:
There are no current labels that start with `npsp`, but interesting there is a very similar one that starts with common: `commonCreditNumber`. So it probably does make sense to just call this one `commonCardholderName`.